.github/workflows/push-manifest.yaml

```diff
@@ -0,0 +1,85 @@
+name: Push Manifest
+on:
+  workflow_call:
+    inputs:
+      source-images:
+        description: 'Space-separated list of source image URLs from build jobs (e.g., "ghcr.io/tenzir/tenzir:main-slim-aarch64 ghcr.io/tenzir/tenzir:main-slim-x86_64")'
+        type: string
+        required: true
+      arch-suffixes:
+        description: 'Space-separated list of architecture suffixes to strip for manifest tags (e.g., "aarch64 x86_64")'
+        type: string
+        required: true
+
+permissions:
+  contents: read
+  id-token: write
+  packages: write
+
+jobs:
+  push-manifest:
+    name: Push Manifest
+    runs-on: ubuntu-latest
+    steps:
+      - name: Ensure XDG_RUNTIME_DIR
+        run: |
+          if [[ -z "$XDG_RUNTIME_DIR" ]]; then
+            XDG_RUNTIME_DIR="/run/user/$(id -u)"
+            mkdir -p "$XDG_RUNTIME_DIR"
+            echo "XDG_RUNTIME_DIR=$XDG_RUNTIME_DIR" >> "$GITHUB_ENV"
+          fi
+      - name: Checkout
```

Comment:
Change the registry login action to use docker login instead of skopeo and remove these steps.